packages/manager/modules/iam/src/components/actionSelect/actionSelect.template.html

```diff
@@ -28,6 +28,36 @@
         <span data-ng-bind-html="$ctrl.customActionSuccessMessage"></span>
     </oui-message>
 
+    <!-- <MANAGED PERMISSIONS> -->
+    <oui-collapsible
+        data-heading="{{ ::'iam_managed_permissions_heading'  | translate }}"
+        data-expanded="true"
+    >
+        <div
+            class="oui-checkbox ml-4"
+            data-ng-repeat="group in $ctrl.permissionsGroupsList track by group.id"
+        >
+            <input
+                class="oui-checkbox__input"
+                id="{{ group.id }}"
+                type="checkbox"
+                name="{{ group.id }}"
+                data-ng-model="group.selected"
+                data-ng-change="$ctrl.onPermissionsGroupsChanged(group.urn)"
+            />
+            <label class="oui-checkbox__label-container" for="{{ group.id }}">
+                <span class="oui-checkbox__label">
+                    <span class="oui-checkbox__icon"></span>
+                    <span class="oui-checkbox__text">
+                        <span data-ng-bind="group.name"></span>
+                    </span>
+                </span>
+                <div class="ml-4" data-ng-bind-html="::group.description"></div>
+            </label>
```

Comment:
To be consistent you can use one time binding everywhere as you've done for `::group.description`

Reply:
Done